app/Wiki.php

```diff
@@ -87,6 +87,11 @@ public function wikiLifecycleEvents(): \Illuminate\Database\Eloquent\Relations\H
         return $this->hasOne(WikiLifecycleEvents::class);
     }
 
+    public function wikibaseNotificationSentRecord(): \Illuminate\Database\Eloquent\Relations\HasOne
+    {
+        return $this->hasOne(WikibaseNotificationSentRecord::class);
```

Comment:
If I understand it correctly, a Wiki can have more than one sent record, so this should be `hasMany`.